opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -394,12 +393,17 @@ message SummaryValue {
 
   // The total number of recorded values since start_time. Optional since
   // some systems don't expose this.
-  google.protobuf.Int64Value count = 3;
+  uint64 count = 3;
 
   // The total sum of recorded values since start_time. Optional since some
   // systems don't expose this. If count is zero then this field must be zero.
-  // This field must be unset if the sum is not available.
-  google.protobuf.DoubleValue sum = 4;
+  double sum = 4;
+
+  // count_unspecified must be set to true if count value is unknown or unspecified.
+  bool count_unspecified = 5;
```

Comment:
curious, isn't `(count == 0) <==> (count_unspecified == true)`? Same for sum.

Reply:
I think those are different assertions:
- `count == 0` means "no observations were made"
- `count_unspecified == true` mean "we don't know how many observations were made"

I don't know how _important_ is the distinction though. In both cases you cannot calculate the average for example.

Also, alternatively the second assertion can be modeled differently. We could assume that:
- if `count==0 && len(percentile_values)==0` then it means "no observations were made"
- if `count==0 && len(percentile_values)>0` then it means "we don't know how many observations were made, but we have data about their distribution"

Could this be good enough and we don't really need to have a distinct condition for unspecified count?

Reply:
I am fine with removing the bools for the moment. If we really need them we can add them back later.

Reply:
@bogdandrutu I like that. Let me do it.

Reply:
Done.